front/src/components/HeaderMenu/BaseSwitcher.tsx

```diff
@@ -0,0 +1,64 @@
+import {
+  Button,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+  Radio,
+  RadioGroup,
+  Stack,
+} from "@chakra-ui/react";
+import { useContext, useState } from "react";
+import { useNavigate } from "react-router-dom";
+
+import { GlobalPreferencesContext } from "providers/GlobalPreferencesProvider";
+
+function BaseSwitcher({ isOpen, onClose }: { isOpen: boolean; onClose: () => void }) {
+  const navigate = useNavigate();
+  const { globalPreferences } = useContext(GlobalPreferencesContext);
+  const currentOrganisationBases = globalPreferences.availableBases;
+  const [value, setValue] = useState("1");
+
+  const switchBase = () => {
+    const currentPath = window.location.pathname.split("/bases/")[1].substring(1);
+
+    navigate(`/bases/${value}${currentPath}`);
+    onClose();
+  };
+
+  return (
+    <>
+      <Modal isOpen={isOpen} onClose={onClose}>
+        <ModalOverlay />
+        <ModalContent>
+          <ModalHeader>Switch Bases to</ModalHeader>
```

Comment:
I don't have access to the designs yet, but because a user can only use the app in one base at a time, it might make sense to change this to "Switch Base to"

Reply:
@aerinsol thoughts?

Reply:
Agree! Typo on my end, will update in designs.

Reply:
@pylipp I didn't manage to reproduce 1. on either Chrome or Firefox:

![msedge_K9csTlEhf5](https://github.com/user-attachments/assets/8fe45896-6c38-4049-8832-7f06f18a8ba2)